assets/lang/en_US.ini

```diff
@@ -596,7 +575,6 @@ Texture Filter = Texture filtering
 Texture Filtering = Texture filtering
 Texture Scaling = Texture scaling
 Texture Shader = Texture shader
-Turn off Hardware Tessellation - unsupported = Turn off "hardware tessellation": unsupported
```

Comment:
Possibly misleading text to be fair, it should probably really be "Ignoring", but anyway this is used in GPU_GLES:

https://github.com/hrydgard/ppsspp/blob/287dd66f20f2a03acd316a6f5d5b98ff2eb6ad26/GPU/GLES/GPU_GLES.cpp#L121

-[Unknown]